cosmo/monitors/dark_monitors.py

```diff
@@ -46,7 +45,7 @@ class FUVALeftDarkMonitor(BaseMonitor):
     name = 'FUVA Dark Monitor - Left'
     data_model = FUVDarkDataModel
     labels = ['ROOTNAME']
-    # output = add path
+    output = # path
```

Comment:
You could fetch this from `SETTINGS` if you'd like. See `acq_monitors.py` or one of the OSM monitor modules for an example.